packages/legend-graph/src/graph/metamodel/pure/test/TestBatch.ts

```diff
@@ -0,0 +1,10 @@
+import type { Hashable } from '@finos/legend-shared';
+import type { TestAssertion } from './assertion/TestAssertion.js';
+
```

Comment:
please add copyright header to this

Reply:
I have a patch for the changeset and license info. Unfortunately, I can't update Chloe's branch (she's Singapore based and not available).

Can I include in a PR immediately after merging this one? Or send one of you a patch to apply onto Chloe's PR?

Reply:
Is this urgent? We don't have plan to release Studio soon.

Reply:
Trying to hit a milestone for Q3 (tomorrow). When's the next release scheduled?

Reply:
@beekemarie FYI -- here's the final PR for the milestone

Reply:
@davidsunglee  I need to do a round of review for this which I can do tomorrow since I brought testable to studio and want to ensure I am comfortable with the changes.
@chloeminkyung could we please ensure the builds are green and add corresponding clips/screenshot for global test runner with a test batch. That is the shared component for all testable. Thanks.

Reply:
@MauricioUyaguari Sounds good -- thanks for the update.

Reply:
Sorry was sick and out of office for few days. will update this by EOD